tools/testbench/file.c

```diff
@@ -743,11 +743,12 @@ static int file_reset(struct comp_dev *dev)
 }
 
 static int file_get_hw_params(struct comp_dev *dev,
-			      struct sof_ipc_stream_params *params)
+			      struct sof_ipc_stream_params *params, int dir)
```

Comment:
Please add a line like this;

Fixes: first12digitsOFgitHash ("Commit subject") above the Signed-off-by line,

Reply:
Added

Reply:
@ktrzcinx  should look like this:
```
Fixes: 2b84838 ("Testbench: Instantiate file component as host")
```

Reply:
Changed